app/components/personal_details_review_component.rb

```diff
@@ -0,0 +1,20 @@
+class PersonalDetailsReviewComponent < ActionView::Component::Base
+  validates :application_form, presence: true
+
+  def initialize(application_form:)
+    @application_form = application_form
+    @personal_details_form = CandidateInterface::PersonalDetailsForm.build_from_application(
+      application_form,
+    )
+  end
+
+  def rows
+    CandidateInterface::PersonalDetailsReviewPresenter
+      .new(@personal_details_form)
+      .present
```

Comment:
Aesthetics: I find `present` here a bit murky because it implies that the `PersonalDetailsReviewPresenter` is doing to be rendering a string. In fact it's going to return an array. Would it make more sense to call that method `rows`?

```
<%= render(SummaryCardComponent, rows: @personal_details_review.rows) %>
```

Reply:
Hmm, that is a better name.